lib/wsdl.js

```diff
@@ -1112,6 +1112,12 @@ WSDL.prototype._initializeOptions = function (options) {
   } else {
     this.options.returnFault = false;
   }
+  if (options.returnNilAsNull !== undefined) {
```

Comment:
@aconran sorry, somehow I managed to oversee your notification 😞 

I'd favor for `handleNilAsNull` as this is describing the scenario 💯 % 😄